augur/frequencies.py

```diff
@@ -139,10 +151,10 @@ def run(args):
             for region in args.regions:
                 # Omit strains sampled prior to the first pivot from frequency calculations.
                 # (these tend to be reference strains included for phylogenetic context)
-                if region=='global':
+                if region==DEFAULT_REGION:
                     node_filter_func = lambda node: node.attr["num_date"] >= pivots[0]
```

Comment:
Pair this with the other suggestion above:

```suggestion
                if not filter_to_region:
                    node_filter_func = lambda node: node.attr["num_date"] >= pivots[0]
```

Reply:
Hmm, this does not work as expected as we are looping through the regions list. We always have to special case the `DEFAULT_REGION` regardless of whether other regions are specified.

Reply:
Oh right! Disregard this then.